testcases/crypto/ssl3_func.c

```diff
@@ -637,6 +637,209 @@ CK_RV do_SSL3_MultipleKeysDerive(CK_SESSION_HANDLE session)
     return rc;
 }
 
+CK_RV do_TLS_PreMasterKeyGen(CK_SESSION_HANDLE session)
+{
+    CK_MECHANISM mech;
+    CK_VERSION version;
+    CK_OBJECT_HANDLE h_key;
```

Comment:
CK_OBJECT_HANDLE h_key = CK_INVALID_HANDLE;   ?

Reply:
well, this function is an exact copy from above, jus the mechanism is changed. I tend to leave it as is, otherwise I would need to change it in all functions in that file.